core/math/a_star.cpp

```diff
@@ -693,7 +693,7 @@ void AStar2D::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("reserve_space", "num_nodes"), &AStar2D::reserve_space);
 	ClassDB::bind_method(D_METHOD("clear"), &AStar2D::clear);
 
-	ClassDB::bind_method(D_METHOD("get_closest_point", "to_position"), &AStar2D::get_closest_point);
+	ClassDB::bind_method(D_METHOD("get_closest_point", "to_position", "ignore_disabled"), &AStar2D::get_closest_point, DEFVAL(true));
```

Comment:
**false**

Reply:
Fixed.

Reply:
ignore/include still seems to be mixed in the current diff?

Reply:
Hmm, yea, let me try this one more time...